server/src/main/java/io/crate/Constants.java

```diff
@@ -30,5 +30,7 @@ public class Constants {
     // Mapping Type that contains table definitions
     public static final String DEFAULT_MAPPING_TYPE = "default";
 
+    public static final String DEFAULT_DATE_STYLE = "ISO";
+
```

Comment:
```suggestion
```

I think exposing this as public constant could be error prone, because once we allow to change the session setting, components would operate on the static value. Instead they must go through the session settings to use the correct value.

Reply:
Should the default value `Supplier` then be part of the `CoordinatorSessionSettings` ?
Otherwise a default value is used both in that `Supplier` and in the constructor for the `CoordinatorSessionSettings`.

Reply:
Good question. I'm not sure either. Let's leave it as is for now, we can follow up on this later.